api/spec/spec_handler.go

```diff
@@ -606,6 +607,12 @@ func (d *specHandler) UpdateSpecFromOpts(opts map[string]string, spec *api.Volum
 			} else {
 				spec.FpPreference = fastpath
 			}
+		case api.SpecWinshare:
+			if winshare, err := strconv.ParseBool(v); err != nil {
+				return nil, nil, nil, err
+			} else {
+				spec.Winshare = winshare
+			}
```

Comment:
else is not needed after return.

Reply:
but that is how code is written in all places IN THIS FUNCTION.